.github/workflows/prometheus-unit-tests.yaml

```diff
@@ -0,0 +1,18 @@
+name: Run prometheus unit tests
+on:
+  pull_request:
+    paths:
+      - 'config/monitoring/prometheus/**'
+      - 'tests/prometheus_unit_tests/**'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Install Promtool
+      run: |
+        apt-get update
```

Comment:
This seems to be failing. I don't know enough about GH actions to know if this is expected behaviour for some reason :thinking:

Reply:
Yeap, I tried a different approach to installing Promtool.

Reply:
Fixed!

Reply:
It is actually two different steps? I mean, installling promtool from ubuntu repos (and it's named "Install Promtool") has nothing to do with actions/checkout which checkouts the repo. @AjayJagan ?

Reply:
My thought was, it's I was following the same format of existing workflows like [this](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/.github/workflows/unit-tests.yaml) where we're breaking out the install and checkout in separate steps. Do you think this is bad idea?

Reply:
That's a small change, I can add the name. No issues.